### ISSUE TITLE
[Bug]: Battery Doorbell already exists lib won't start

### ISSUE THREAD

**Issue by lenoxys** [bropat/eufy-security-client#145]
### Client version

2.0.0

### Node version

16.15

### Operating System type

Linux

### Operating system version

Debian 11

### Describe the bug

Hello, 

When starting the script and getting the list of device the lib will stop when trying to load the battery doorbell.

I was looking in insane debug mode but I don't see anything related to that issue. lib crash right away loading the doorbell.

Regards,
Aurelien

### To reproduce

Have a battery doorbell seems to make the issue

### Screenshots & Logfiles

```
DuplicateDeviceError: Device with this serial T8210PXXXXXXXXX exists already and couldn't be added again!
    at EufySecurity.addDevice (/home/adelamarre/plugin2/node_modules/eufy-security-client/src/eufysecurity.ts:285:19)
    at EufySecurity.handleDevices (/home/adelamarre/plugin2/node_modules/eufy-security-client/src/eufysecurity.ts:491:22)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by bropat** [comment]
@lenoxys 

Could you please send me the complete debug logs?

**Comment by lenoxys** [comment]
Can I send it to you by email ?

**Comment by bropat** [comment]
> Can I send it to you by email ? 

Yeah

**Comment by lenoxys** [comment]
> > Can I send it to you by email ?
> 
> Yeah

done

**Comment by bropat** [comment]
@lenoxys 

I replied to your email and attached a test version that should fix the problem.
Could you please try it and give me feedback?

Thx!

**Comment by lenoxys** [comment]
Hey,

looking at my email. I didn't see anything even on my spam. Is it the one in develop branch ?

Thx!

**Comment by bropat** [comment]
Gmail has blocked the mail... :(

[eufy-security-client-2.0.1.zip](https://github.com/bropat/eufy-security-client/files/8604152/eufy-security-client-2.0.1.zip)

Rename it from .zip to .tgz

**Comment by bropat** [comment]
Fixed in version 2.0.1
